Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use the Python 3.11 slim image as the base image
+FROM python:3.11-slim
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Enable non-interactive mode for debconf
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Update the package repository and install required dependencies
+RUN apt-get update -y && \
+    apt-get install -y build-essential cmake unzip pkg-config \
+    libjpeg-dev libpng-dev libtiff-dev \
+    libavcodec-dev libavformat-dev libswscale-dev libv4l-dev \
+    libatlas-base-dev gfortran && \
+    apt-get clean && \
+    apt-get -y autoremove
+
+# Install Python dependencies
+RUN pip3 install --user opencv-python opencv-contrib-python
```

Comment:
Can you please test it with `opencv-python-headless opencv-contrib-python-headles` once? That should make it more optimised

Reply:
Added headless version working perfectly.